app/src/main/java/com/github/fi3te/notificationcron/ui/NotificationCronDialog.kt

```diff
@@ -45,11 +47,12 @@ fun showUpdateDialog(
         setInput(this, value)
 
         positiveButton(R.string.update) {
-            val (cron, notificationTitle, notificationText) = getInput(this)
+            val (cron, notificationTitle, notificationText, onClickURI) = getInput(this)
```

Comment:
For a better readability I would prefer onClickUri

Reply:
Similar to https://github.com/fi3te/NotificationCron/pull/19#discussion_r532101127